templates/terraform/constants/spanner_database.go.erb

```diff
@@ -0,0 +1,45 @@
+<%- # the license inside this block applies to this file
+	# Copyright 2020 Google Inc.
+	# Licensed under the Apache License, Version 2.0 (the "License");
+	# you may not use this file except in compliance with the License.
+	# You may obtain a copy of the License at
+	#
+	#     http://www.apache.org/licenses/LICENSE-2.0
+	#
+	# Unless required by applicable law or agreed to in writing, software
+	# distributed under the License is distributed on an "AS IS" BASIS,
+	# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	# See the License for the specific language governing permissions and
+	# limitations under the License.
+-%>
+// customizeDiff func for additional checks on google_spanner_database properties:
+func resourceSpannerDBDdlCustomDiffFunc(diff TerraformResourceDiff) error {
+	old, new := diff.GetChange("ddl")
+	oldDdls := old.([]interface{})
+	newDdls := new.([]interface{})
+	var err error
+
+	if len(newDdls) < len(oldDdls) {
+		err = diff.ForceNew("ddl")
+		if err != nil {
+			return fmt.Errorf("ForceNew failed for ddl, old - %s and new - %s", oldDdls, newDdls)
```

Comment:
Thanks for adding the error handling! I think in this case since oldDdls and newDdls are arrays, that you'll want to use a format modifier like %v rather than %s, which is for strings.